src/app/pages/Lend/views/LoanProposalsTable/LoanProposalsTable.tsx

```diff
@@ -110,15 +113,21 @@ const makeColumns = (backgroundColor: string): Array<NewTable.models.Column<Loan
     align: 'right',
     cellContent: {
       kind: 'simple',
-      render: () => (
-        <Button
-          variant="outlined"
-          color="primary"
-          onClick={() => undefined}
-          backgroundColor={backgroundColor}
-        >
-          Stake
-        </Button>
+      render: x => (
+        <>
+          {!x.isOwnProposal && (
+            <div style={{ display: 'inline-flex' }}>
+              <GivingStakeButton
```

Comment:
Remove inline wrapper, go to src/features/giveStake/GivingStakeButton.tsx, and remove fullWidth property from ModalButton.